manifests/init.pp

```diff
@@ -199,99 +199,119 @@
-  Enum['debug', 'info', 'warn', 'error', 'fatal'] $logging_level = $foreman::params::logging_level,
-  Enum['file', 'syslog', 'journald'] $logging_type = $foreman::params::logging_type,
-  Enum['pattern', 'multiline_pattern', 'multiline_request_pattern', 'json'] $logging_layout = $foreman::params::logging_layout,
-  Hash[String, Boolean] $loggers = $foreman::params::loggers,
-  Optional[Enum['sendmail', 'smtp']] $email_delivery_method = $foreman::params::email_delivery_method,
-  Optional[Stdlib::Host] $email_smtp_address = $foreman::params::email_smtp_address,
-  Stdlib::Port $email_smtp_port = $foreman::params::email_smtp_port,
-  Optional[Stdlib::Fqdn] $email_smtp_domain = $foreman::params::email_smtp_domain,
-  Enum['none', 'plain', 'login', 'cram-md5'] $email_smtp_authentication = $foreman::params::email_smtp_authentication,
-  Optional[String] $email_smtp_user_name = $foreman::params::email_smtp_user_name,
-  Optional[String] $email_smtp_password = $foreman::params::email_smtp_password,
-  String $telemetry_prefix = $foreman::params::telemetry_prefix,
-  Boolean $telemetry_prometheus_enabled = $foreman::params::telemetry_prometheus_enabled,
-  Boolean $telemetry_statsd_enabled = $foreman::params::telemetry_statsd_enabled,
-  String $telemetry_statsd_host = $foreman::params::telemetry_statsd_host,
-  Enum['statsd', 'statsite', 'datadog'] $telemetry_statsd_protocol = $foreman::params::telemetry_statsd_protocol,
-  Boolean $telemetry_logger_enabled = $foreman::params::telemetry_logger_enabled,
-  Enum['DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'] $telemetry_logger_level = $foreman::params::telemetry_logger_level,
-  Boolean $dynflow_manage_services = $foreman::params::dynflow_manage_services,
-  Enum['present', 'absent'] $dynflow_orchestrator_ensure = $foreman::params::dynflow_orchestrator_ensure,
-  Integer[0] $dynflow_worker_instances = $foreman::params::dynflow_worker_instances,
-  Integer[0] $dynflow_worker_concurrency = $foreman::params::dynflow_worker_concurrency,
-  Optional[Redis::RedisUrl] $dynflow_redis_url = $foreman::params::dynflow_redis_url,
-  Boolean $hsts_enabled = $foreman::params::hsts_enabled,
-  Array[Stdlib::HTTPUrl] $cors_domains = $foreman::params::cors_domains,
-  Integer[0] $foreman_service_puma_threads_min = $foreman::params::foreman_service_puma_threads_min,
-  Integer[0] $foreman_service_puma_threads_max = $foreman::params::foreman_service_puma_threads_max,
-  Integer[0] $foreman_service_puma_workers = $foreman::params::foreman_service_puma_workers,
-  Hash[String, Any] $rails_cache_store = $foreman::params::rails_cache_store,
-  Boolean $keycloak = $foreman::params::keycloak,
-  String[1] $keycloak_app_name = $foreman::params::keycloak_app_name,
-  String[1] $keycloak_realm = $foreman::params::keycloak_realm,
+  Stdlib::HTTPUrl                                                           $foreman_url                      = $foreman::params::foreman_url,
```

Comment:
Could you un-align it in all files again?

Reply:
Took me some Time with `git rebase -i` while keeping the Commits separate.  But now I unaligned all Files